src/main/java/com/leverx/learningmanagementsystem/btp/registry/controller/SubscriptionController.java

```diff
@@ -0,0 +1,53 @@
+package com.leverx.learningmanagementsystem.btp.registry.controller;
```

Comment:
I suggest to place this controller in package `com.leverx.learningmanagementsystem.multitenancy.subscription.controller` (and everything related to multitenancy here)
As it's not explicitly related to SaaS Provisioning service, this API can be triggered by anyone who has `Callback` scope, SubscriptionService can be implemented in different ways based on environment where tenant should be subscribed